Cargo.toml

```diff
@@ -33,7 +33,7 @@ jwt-simple = { version = "0.12", default-features = false, features = [
     "pure-rust",
 ] }
 kbs_protocol = { git = "https://github.com/confidential-containers/guest-components.git", rev = "e6999a3c0fd877dae9e68ea78b8b483062db32b8", default-features = false }
-kbs-types = "0.7.0"
+kbs-types = { git = "https://github.com/virtee/kbs-types.git", rev = "a704036" }
```

Comment:
looks like `ProtectedHeader::generate_aad()` is not released yet. Will we do another kbs-types release before we merge this?

Reply:
I think it is ok to use a rev here. Once we have another release we can update then.